src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -896,4 +896,8 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>Package content hash validation failed for {0}. Expected: {1} Actual: {2}</value>
     <comment>0 - package ID, 1 - expected Sha, 2 - actual sha</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="Error_InvalidProjectRestoreStyle" xml:space="preserve">
+    <value>Invalid project restore style '{0}'.</value>
```

Comment:
We should probably add this validation in VS as well. 

Not to do it in this PR but maybe create a tracking issue for it.